src/sentry/static/sentry/app/components/events/eventEntries.tsx

```diff
@@ -290,14 +316,21 @@ const BorderlessEventEntries = styled(EventEntries)`
   }
 `;
 
+type StyledEventUserFeedbackProps = React.ComponentProps<typeof EventUserFeedback> & {
+  includeBorder: boolean;
+  theme?: any;
+};
+
 const StyledEventUserFeedback = styled(EventUserFeedback)`
```

Comment:
If you do 
```suggestion
const StyledEventUserFeedback = styled(EventUserFeedback)<StyledEventUserFeedbackProps>`
```
I don't think you'll need to type the props in each lambda in the CSS template string.